src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -4,6 +4,15 @@
 import ansys.platform.instancemanagement as pypim
 
 
+class PyPIMConfigurationError(ConnectionError):
+    """Provides the error when `PyPIM<https://pypim.docs.pyansys.com/version/stable/>` is not configured."""
+
+    def __init__(self):
+        super().__init__(
+            "Verify the value of the 'ANSYS_PLATFORM_INSTANCEMANAGEMENT_CONFIG' environment variable."
```

Comment:
We don't need to give advice on details of setting up pim - that's a maintenance burden.

Reply:
Right, we will update this as "PyPIM is not configured."

Reply:
Done. Thank you.